discovery.tf

```diff
@@ -26,3 +31,70 @@ resource "google_project_iam_member" "discovery_run_viewer" {
   role    = "roles/run.viewer"
   member  = "serviceAccount:${google_service_account.chainguard_discovery.email}"
 }
+
+
+///////////////////////////////////////////////////////////////////////////
+//
+// Configure a pubsub topic audit log sink, so that we can subscribe to
+// updates in each of the services that we support discovering and
+// monitoring.
+//
+///////////////////////////////////////////////////////////////////////////
+resource "google_pubsub_topic" "gcp_auditlog_sink" {
+  name = "chainguard-auditlogs"
+}
+
+// Allow our discovery identity to manage and consume subscriptions in the
+// project containing the topic and service accounts.
+// See also: https://groups.google.com/g/cloud-pubsub-discuss/c/DhfkLMWcRok
+resource "google_project_iam_custom_role" "create_auditlog_subscriptions" {
+  role_id     = "chainguardSubscriber"
+  title       = "Chainguard Subscriber"
+  description = "This role allows certain Chainguard roles to manage pub/sub subscriptions."
```

Comment:
nit: s/This role allows certain Chainguard roles/This role allows certain Chainguard service accounts/

Reply:
https://github.com/chainguard-dev/terraform-google-chainguard-account-association/pull/16